test/unittest/unittest_nntrainer_tensor_neon_fp16.cpp

```diff
@@ -1571,6 +1571,38 @@ TEST(nntrainer_Tensor, inv_sqrt_i_p) {
   EXPECT_EQ(flag, true);
 }
 
+/**
+ * @brief fp16 tensor has NaN
+ */
+TEST(nntrainer_Tensor, is_valid_01) {
+  size_t batch = 1;
+  size_t channel = 3;
+  size_t height = 4;
+  size_t width = 5;
+
+  nntrainer::Tensor input(
+    {batch,
+     channel,
+     height,
+     width,
+     {nntrainer::Tformat::NCHW, nntrainer::Tdatatype::FP16}},
+    true, nntrainer::Tensor::Initializer::ZEROS);
```

Comment:
As far as I know, this part has been changed to the following.

```nntrainer::Tensor::Initializer::ZEROS); ``` --> ```nntrainer::Initializer::ZEROS```

Reply:
Fixed. Thanks!